packages/PoppyPrint-Core.package/PPFormatter.class/instance/evaluateBraceStatements.multiLine..st

```diff
@@ -0,0 +1,16 @@
+helper
+evaluateBraceStatements: aNode multiLine: multiLine
+
+	aNode elements do: [:element | | isLast |
+		isLast := element = aNode elements last.
+		((self isOptionalNewLineMarker: element) and: multiLine) ifFalse: [
+			((self isEmptyLineMarker: element) or: (self isOptionalEmptyLineMarker: element))
+				ifTrue: [self newLine]
+				ifFalse: [
+					((self isEmptyLineMarker: element) or: (self isOptionalNewLineMarker: element)) ifFalse: [
+						self visitNode: element.
+						isLast ifFalse: [
+							config spaceBeforePointInArray ifTrue: [stream space].
+							self stream nextPut: $.]].
+					
+					isLast ifFalse: [multiLine ifTrue: [self newLine] ifFalse: [self stream space]]]]]
```

Comment:
Same here
- `statement1 or: [statement2]` not `statement1 or: statement2`
- `statement1 and: [statement2]` not `statement1 and: statement2`